qa/integration-tests/src/test/java/io/zeebe/broker/it/startup/BrokerReprocessingTest.java

```diff
@@ -168,19 +168,16 @@ public void shouldContinueWorkflowInstanceAtTaskAfterRestart() {
         .send()
         .join();
 
-    assertJobCreated("foo");
+    final var jobFoo = RecordingExporter.jobRecords(JobIntent.CREATED).withType("foo").getFirst();
 
     // when
     reprocessingTrigger.accept(this);
-
     clientRule
         .getClient()
-        .newWorker()
-        .jobType("foo")
-        .handler(
-            (client, job) ->
-                client.newCompleteCommand(job.getKey()).variables(NULL_VARIABLES).send())
-        .open();
+        .newCompleteCommand(jobFoo.getKey())
```

Comment:
wait... doesn't the job have to be `ACTIVATED` first?

Reply:
Nope this is also what TaskList for example does :)

Reply:
mind is 🤯